modules/alacritty/hm.nix

```diff
@@ -2,63 +2,71 @@
 # - https://alacritty.org/config-alacritty.html
 # - `man 5 alacritty`
 { config, lib, ... }:
-
-let
-  colors = config.lib.stylix.colors.withHashtag;
-in
-{
-  options.stylix.targets.alacritty.enable =
-    config.lib.stylix.mkEnableTarget "Alacritty" true;
-
-  config =
-    lib.mkIf (config.stylix.enable && config.stylix.targets.alacritty.enable)
+lib.modules.importApply ../../stylix/mk-target.nix {
```

Comment:
I'd recommend not using `importApply` for `mk-target`, because I assume we want the target options to show as declared by the calling module (i.e. `modules/alacritty/hm.nix`) _not_ as declared by `stylix/mk-target.nix`.

E.g. using `importApply` would cause most of alacritty's options to show as declared by `stylix/mk-target.nix` in the docs and link to that page instead of to here.

Reply:
(same underlying issue as https://github.com/danth/stylix/pull/1130#discussion_r2072680408)